Sources/CustomerCenter/CustomerCenterConfigData.swift

```diff
@@ -250,13 +269,17 @@ extension CustomerCenterConfigData.Appearance.AppearanceMode {
         case .system:
             self = .system
         case .custom:
+            func parseColorInformation(from response: CustomerCenterConfigResponse.Appearance) throws -> CustomerCenterConfigData.Appearance.ColorInformation {
+                return CustomerCenterConfigData.Appearance.ColorInformation(
+                    light: try RCColor(stringRepresentation: response.light.accentColor),
+                    dark: try RCColor(stringRepresentation: response.dark.accentColor)
```

Comment:
This is always using the `accentColor` but the callers are also expecting background/text colors right?